### PR TITLE
Reactivation to book coupon

### PR DESCRIPTION
**Pull request by debck**
#2 
I made the following changes. Not sure what to do next @knrt10

### CODE REVIEW

**Comment by debck** [comment]
@knrt10  is it correct or i have to make changes?

**Comment by knrt10** [comment]
Will try locally today.

File: src/routes/adminRoutes.ts [hunk 1]
```diff
@@ -26,6 +27,15 @@ export class AdminRoutes extends BaseRoutes {
     this.router.route("/mess1/all/:id").get(isAdminOrUser, (req: Authenticated, res) => this.getUsersForGroundMess(req, res));
     this.router.route("/mess2/all/:id").get(isAdminOrUser, (req: Authenticated, res) => this.getUsersForUpperMess(req, res));
     this.router.route("/god/:id").post((req, res) => this.dropCollection(req, res));
+
+    //Clean Database every Sunday
+    cron.schedule('* * * * Sunday', () => {
```
Comment (knrt10) [inline src/routes/adminRoutes.ts]: @debck as the server is hosted on some foreign country so time won't be scheduled as per our IST. So we need to convert it according to IST i.e we can use in UTC format.

So change the code to this after importing **moment**

```ts
const presentDay = moment().utcOffset("+05:30").format("dddd");
    if (presentDay === "Monday") {
      cron.schedule("0 0 0 * * Monday", () => {
        console.log("running a task every Monday");
        Coupon.find({}).then((coupon: any) => {
          if (coupon.length !== 0) {
            Coupon.collection.drop();
            console.log("Database Deleted");
          }
          else {
            console.log("Already deleted database")
          }
        });
      });
    }
```

**Review by knrt10 — CHANGES_REQUESTED** [review]
@debck please change accordingly

**Comment by debck** [comment]
@knrt10 made changes ...plz check it

**Review by knrt10 — APPROVED** [review]
@debck cool

**Pull request merged**